css/utils/vars.css

```diff
@@ -2,4 +2,5 @@
   --primary--color: #6d4930;
   --text--primary--color: white;
   --nav--input--border: black;
+  --centered--box-color: white;
```

Comment:
try to use hex or rgba values(cause different browsers have different shades of the color)